service-catalog/apis/grafana.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: grafana
+  description: Public grafana api of our operate first instance
+spec:
+  type: openapi
+  lifecycle: production
+  owner: group:operate-first
+  system: grafana
+  definition:
+    $text: https://github.com/grafana/grafana/blob/main/public/api-spec.json
```

Comment:
It worked even like this, and I think it may be using the GitHub OAuth login to get the data. But I agree with you that it should be `raw` link, however this needs a new addition to the allowed hosts in the `app-config.yaml`.